DOCUMENTATION.md

```diff
@@ -1034,3 +1342,70 @@ This section provides methods and functionalities designed to interact with the
 - **Arguments:**
   - `directRequestConsumerAddress`: Direct Request Consumer address
 
+
+
+### Service alias: `functionsConsumer`
```

Comment:
As per other comments, the Function Consumer is not a service or Interface.  It is just an example contract.  The user can use any smart contract they like as long as it inherits `FunctionsClient` and implements the `fulfilRequest`() method in the specified signature.

Reply:
To make it possible to run Functions Simulations it is not enough to implement only `fulfillRequest()`, `sendRequest()` should be also available to call.